### ISSUE TITLE
Maintenance in 2023

### ISSUE THREAD

**Issue by Prakalpit** [bitlabstudio/django-calendarium#68]
Is this project being maintained in 2023, I can see not much changes have been made over the years, is it still good to use it in 2023?

**Comment by mbrochh** [comment]
No, we no longer maintain any of our projects. If you would like to take over ownership, please let me know.
